pkg/command/plugin_search.go

```diff
@@ -82,7 +82,22 @@ func newSearchPluginCmd() *cobra.Command {
 	f.BoolVar(&showDetails, "show-details", false, "show the details of the specified plugin, including all available versions")
 	f.StringVarP(&pluginName, "name", "n", "", "limit the search to plugins with the specified name")
 	f.StringVarP(&outputFormat, "output", "o", "", "output format (yaml|json|table)")
-	f.StringVarP(&local, "local", "l", "", "path to local plugin source")
+
+	f.StringVarP(&local, "local", "", "", "path to local plugin source")
+	msg := fmt.Sprintf("this was done in the %q release, it will be removed following the deprecation policy (6 months). Use the %q flag instead.\n", "v1.0.0", "--local-source")
```

Comment:
nit:
Perhaps the construction of the message could be factored out into a reusable helper function to encourage some consistency in the deprecation message.

Reply:
Good point. Will do.